roles/ceph-common/defaults/main.yml

```diff
@@ -1,107 +1,107 @@
-  osd_scrub_chunk_max: 5
-  osd_deep_scrub_stride: 1048576
+---
+ceph_common:
+
+  ###########
+  # INSTALL #
+  ###########
+
+  debian_pkgs:
+    - python-pycurl
+    - ntp
+    - hdparm
+    - libvirt-bin
+
+  ceph_pkgs:
+    - ceph
+    - ceph-common
+
+  stable_release: hammer
+
+  ######################
+  # CEPH CONFIGURATION #
+  ######################
+
+  ## Ceph options
+  #
+  cephx: true
+  cephx_require_signatures: true # Kernel RBD does NOT support signatures for Kernels < 3.18!
+  cephx_cluster_require_signatures: true
+  cephx_service_require_signatures: false
+  max_open_files: 131072
+  disable_in_memory_logs: true # set this to false while enabling the options below
+
+  # Debug logs
+  enable_debug_global: false
+  debug_global_level: 20
+  enable_debug_mon: false
+  debug_mon_level: 20
+  enable_debug_osd: false
+  debug_osd_level: 20
+  enable_debug_mds: false
+  debug_mds_level: 20
+
+  ## Client options
+  #
+  rbd_cache: "true"
+  rbd_cache_writethrough_until_flush: "true"
+  rbd_concurrent_management_ops: 20
+  rbd_client_directories: false # this will create rbd_client_log_path and rbd_client_admin_socket_path directories with proper permissions, this WON'T work if libvirt and kvm are installed
+  rbd_client_log_file: /var/log/rbd-clients/qemu-guest-$pid.log # must be writable by QEMU and allowed by SELinux or AppArmor
+  rbd_client_log_path: /var/log/rbd-clients/
+  rbd_client_admin_socket_path: /var/run/ceph/rbd-clients # must be writable by QEMU and allowed by SELinux or AppArmor
+  rbd_default_features: 3
+  rbd_default_map_options: rw
+  rbd_default_format: 2
+
+  ## Monitor options
+  #
+  monitor_interface: bond0
+  mon_osd_down_out_interval: 600
+  mon_clock_drift_allowed: .15
+  mon_clock_drift_warn_backoff: 30
+  mon_osd_full_ratio: .95
+  mon_osd_nearfull_ratio: .70
+  mon_osd_report_timeout: 300
+  mon_pg_warn_max_per_osd: 0 # disable complains about low pgs numbers per osd
+  mon_osd_allow_primary_affinity: "true"
+
+  ## OSD options
+  #
+  journal_size: 10000 # 10 GB
+  pool_name: default
+  pool_default_size: 3
+  pool_default_min_size: 2
+  osd_mkfs_type: xfs
+  osd_mkfs_options_xfs: -f -i size=2048
+  osd_mount_options_xfs: noatime,largeio,inode64,swalloc
+  osd_mon_heartbeat_interval: 30
+  osd_backfill_full_ratio: .95
+
+  # CRUSH
+  pool_default_crush_rule: 0
+  osd_crush_update_on_start: "true"
+
+  # Object backend
+  osd_objectstore: filestore
+
+  # Performance tuning
+  filestore_merge_threshold: 40
+  filestore_split_multiple: 8
+  osd_op_threads: 8
+  filestore_op_threads: 8
+  filestore_max_sync_interval: 5
+  osd_max_scrubs: 1
+
+  # Recovery tuning
+  osd_recovery_max_active: 1
+  osd_max_backfills: 1
+  osd_recovery_op_priority: 2
+  osd_recovery_max_chunk: 1048576
+  osd_recovery_threads: 1
+
+  # Deep scrub
+  osd_scrub_sleep: .1
+  osd_disk_thread_ioprio_class: idle
+  osd_disk_thread_ioprio_priority: 0
+  osd_scrub_chunk_max: 5
+  osd_deep_scrub_stride: 1048576
```

Comment:
nothing has changed in this file, other than the new line characters were ^M, which is not in the openstack style guidelines http://docs.openstack.org/developer/hacking/